lxd/instance/drivers/driver_lxc.go

```diff
@@ -7913,8 +7938,46 @@ func (d *lxc) removeDiskDevices() error {
 	return nil
 }
 
-// Network I/O limits.
-func (d *lxc) setNetworkPriority() error {
+func setNetworkPriorityWithNetfilter(d *lxc, networkPriority int) error {
+	// in setNetworkPriorityWithCgroup we set skb->priority on all
+	// skb's that comes from the container cgroup on the enter to
+	// the host interfaces
+	// here we go another way, we set skb->priority on all skb's
+	// that exits the container network namespace once skb exits
+	// from the host peer of the container network device.
+
+	// Check that we at least succeeded to set an entry
+	success := false
+	var lastError error
+	for devName, devConfig := range d.ExpandedDevices() {
+		if devConfig["type"] != "nic" {
+			continue
+		}
+
+		hostDevName := d.ExpandedConfig()[fmt.Sprintf("volatile.%s.host_name", devName)]
+
+		var err error
+
+		err = d.state.Firewall.InstanceClearNetPrio(d.Project().Name, d.Name(), hostDevName)
```

Comment:
```suggestion
		err := d.state.Firewall.InstanceClearNetPrio(d.Project().Name, d.Name(), hostDevName)
```

This can be simplified a bit.